docs/page-api.md

```diff
@@ -10,7 +10,7 @@ A binding of URL paths to a Page class that can render the results for those URL
 
 * The HTTP method(s) that are acceptable.
 
-* Default: “get”
+* Default: [“get”, "head"]
```

Comment:
As it stands right now, `react-server` throws a 404 error for all HEAD requests because routes being passed to `routr` don't send the `method` attribute with it.  Therefore, `routr` defaults to 'GET' and checks the request.  `head !== get`, therefore there's no route found for HEAD/POST/PUT/PATCH/etc.